enhancements/assets-generation/cloud-foundry-application-discovery/README.md

```diff
@@ -300,35 +300,33 @@ expectations).
 
 ### Upgrade / Downgrade Strategy
 
-If applicable, how will the component be upgraded and downgraded? Make sure this
-is in the test plan.
-
-Consider the following in developing an upgrade/downgrade strategy for this
-enhancement:
-- What changes (in invocations, configurations, API use, etc.) is an existing
-  cluster required to make on upgrade in order to...
-  -  keep previous behavior?
-  - make use of the enhancement?
+N/A
 
 ## Implementation History
 
-Major milestones in the life cycle of a proposal should be tracked in `Implementation
-History`.
+- January 2025: Proposal created and approved.
+- (Tentative) February-March 2025: Initial MVP with support for CF manifest located in the filesystem where the CLI runs.
+  Some limitations might apply based on the use cases to implement first.
+- (Tentative) April 2025: Planned tech-preview release.
 
 ## Drawbacks
 
-The idea is to find the best form of an argument why this enhancement should _not_ be implemented.
+- M2K already provides discovery for Cloud Form application manifests. We could extend
+  it to reuse the discovery logic and avoid the cost of redoing what is already provided.
+- Breaks with existing M2K CLI support for CF application discovery. Some existing discovery
+  functionality might not be supported in the short term and users will have to migrate their
+  templates to the new Helm based template engine.
 
 ## Alternatives
 
-Similar to the `Drawbacks` section the `Alternatives` section is used to
-highlight and record other possible approaches to delivering the value proposed
-by an enhancement.
+- Reuse the existing M2K functionality for discovery, at the expense of using a code that is not ideal to manage
+  and we are not a principal stakeholder.
 
 ## Infrastructure Needed [optional]
 
-Use this section if you need things from the project. Examples include a new
-subproject, repos requested, github details, and/or testing infrastructure.
-
-Listing these here allows the community to get the process for these resources
-started right away.
+- CI/CD pipelines for building and unit/integration testing.
+- CI/CD pipelines for E2E testing for QE and releasing.
+- Hosting provider for binaries and documentation for releases.
+- Project Management tools for tasks, issues and bugs.
+- If REST API discovery is implemented, a Cloud Foundry instance for E2E testing with a suite of samples that cover
```

Comment:
we can use Korifi, maybe a CF instance isn't necessary. WDYT?

Reply:
yeah, as long as we can use something that implements the V3 REST API we're good. I'll update the doc 😄

Reply:
Done, replaced CF with a Korifi instance